spec/support/seeds/seeds_e2e.rb

```diff
@@ -92,7 +92,7 @@
   organization = Organization.create!(
     name: "Bachmanity",
     summary: Faker::Company.bs,
-    remote_profile_image_url: logo = Faker::Company.logo,
+    profile_image: logo = File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
```

Comment:
I faced problems with using remote url (404 errors while setting up), and noticed that `File.open` is used in other seeds. So I supposed that it's ok to replace remote URL with using the local file.